### ISSUE TITLE
Using ConfigClass inside INSTALLED_APPS causes clickhouse_migrate failure

### ISSUE THREAD

**Issue by M1ha-Shvn** [carrotquest/django-clickhouse#34]
> Hi guys, in [this PR](https://github.com/carrotquest/django-clickhouse/pull/33) I've implemented a full featured django `clickhouse_migrate` command for `manage.py`. As soon as tests pass, I'll make a release. You can read more about command [here](https://github.com/carrotquest/django-clickhouse/blob/migrate-command/issue-31/docs/migrations.md#admin-migration-command)

@M1ha-Shvn  It crashes when you install an app by it's config.

```
    spam_spec = find_spec(module_name)
  File "/Users/simion/.pyenv/versions/3.8.5/lib/python3.8/importlib/util.py", line 94, in find_spec
    parent = __import__(parent_name, fromlist=['__path__'])
ModuleNotFoundError: No module named 'dashboards.apps.DashboardsConfig'; 'dashboards.apps' is not a package
```

that's how i added the app:

```python
INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    #...
    'dashboards.apps.DashboardsConfig',  # here
    'widgets.apps.WidgetsConfig',
]
```

However, if i specify just one app for migration, it works:
```
./manage.py clickhouse_migrate dashboard
```

_Originally posted by @simion in https://github.com/carrotquest/django-clickhouse/issues/31#issuecomment-930074245_

**Comment by M1ha-Shvn** [comment]
The reason is that `DashboardConfig` points directly to the `AppConfig` class, while migrate command expects `INSTALLED_APPS` elements to be python module paths. I didn't even know, that it can be used so. I'll have a look around this problem soon. 
Here are [django docs](https://docs.djangoproject.com/en/3.2/ref/applications/#configuring-applications) for future me.

P. s. if you know how to import `INSTALLED_APPS` modules in more correct way, I'll be greatful for a PR. I guess, it should use [application registry](https://docs.djangoproject.com/en/3.2/ref/applications/#application-registry) instead of direct `INSTALLED_APPS` setting in some way...

P. p. s. The problem is in one of these lines:
1. https://github.com/carrotquest/django-clickhouse/blob/master/src/django_clickhouse/management/commands/clickhouse_migrate.py#L31
2. https://github.com/carrotquest/django-clickhouse/blob/master/src/django_clickhouse/migrations.py#L72
3. https://github.com/carrotquest/django-clickhouse/blob/master/src/django_clickhouse/migrations.py#L74

**Comment by M1ha-Shvn** [comment]
Fixed [here](https://github.com/carrotquest/django-clickhouse/pull/35)

**Comment by M1ha-Shvn** [comment]
Released [v1.1.1](https://github.com/carrotquest/django-clickhouse/releases/tag/v1.1.1)
